<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesLanguage</class>
 <widget class="QWidget" name="PreferencesLanguage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="language_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>UI language</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="languageSystemDefault">
        <property name="text">
         <string>System default</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="languageDropdown"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>203</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="languageFilter"/>
      </item>
     </layout>
    </widget>
   </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>languageSystemDefault</sender>
   <signal>toggled(bool)</signal>
   <receiver>languageDropdown</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>languageSystemDefault</sender>
   <signal>toggled(bool)</signal>
   <receiver>languageFilter</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
